lib/integration/queryManager.ts

```diff
@@ -0,0 +1,21 @@
+import { logMessage } from "../logger";
+import { QueryArrayResult, Client } from "pg";
+
+const queryManager = {
+  async executeQuery(
+    client: Client,
+    sql: string,
+    params: [string]
+  ): Promise<QueryArrayResult<unknown[]>> {
+    try {
+      if (!sql) throw new Error("Empty or invalid sql passed as a parameter");
+      //Establish a connexion
+      await client.connect();
```

Comment:
I would assume that at this point a connection as been established so you can run just queries. Leave the handling of the client outside this function

Reply:
that means bringing back  `await client.connect()` inside dbConnector.ts